CHANGELOG.md

```diff
@@ -19,6 +19,12 @@
 
 </details>
 
+## 0.86.0 (December 17, 2018)
```

Comment:
@Jimmy-C We should not have to add the version number in this PR (only happens on release)